docs/content/tutorial/step_03.ngdoc

```diff
@@ -127,7 +134,7 @@ acquired skill.
 <body>
 
   <!-- Use a custom component to render a list of phones -->
-  <phone-list></phone-list>
+  `<phone-list></phone-list>`  <!-- This tells AngularJS to instantiate a phoneList component here. -->
```

Comment:
Sorry, I meant wrap `phoneList`, not `<phone-list></phone-list>`, in backticks :grin:

Reply:
alright fixed, thanks for the info/tips.